a single page website for the upcoming 2019 summit

neta-kedem opened the pull request:
just a simple "coming soon" page

rantav commented on client/static/template.html:
```diff
@@ -4,17 +4,31 @@
     <meta name="viewport" content="width=device-width,initial-scale=1,shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
+    <meta property="og:type" content="article">
+    <meta property="og:title" content="Reversim Summit 2018">
```
Change all these to `2019`?
```suggestion
    <meta property="og:title" content="Reversim Summit 2019">
```
etc

rantav commented on client/.env.development:
```diff
@@ -1,3 +1,3 @@
-HOST=local.reversim.com
```
The reason it is `local.reversim.com` is for google auth to work when running locally. 
Add to your hosts file:

```
$  cat /etc/hosts
...
127.0.0.1 local.reversim.com.
...
```

Not required for now, but you'll need it later

neta-kedem commented:
I can't do a pr to a non existing branch...
I removed pretty much everyting, and left with simple html, css, and some photos and icons (so many icons...).
And changed everyting to 2019

rantav commented:
@neta-kedem I just created an (almost) empty branch https://github.com/reversim/reversim-summit-2019/tree/gh-pages so you can open a PR to this branch (I don't think it's a good idea to merge this only master). 

BTW you're an admin on this repo.
